diff -up 0.4/gx_transform.h~ 0.4/gx_transform.h
--- 0.4/gx_transform.h~	2005-01-10 21:22:55.000000000 +0100
+++ 0.4/gx_transform.h	2015-05-16 12:18:01.631068171 +0200
@@ -3,16 +3,16 @@
 
 
 
-inline __u32 color_unpack_rgb565 (__u32 X);
-inline __u32 color_unpack_rgba4 (__u32 X);
-inline __u32 color_unpack_rgba6 (__u32 X);
-inline __u32 color_unpack_rgb555 (__u32 X);
-inline __u32 color_unpack_rgb4a3 (__u32 X);
-inline __u32 color_unpack_rgb5a3 (__u32 X);
-inline __u32 color_unpack_i4 (__u32 X);
-inline __u32 color_unpack_i8 (__u32 X);
-inline __u32 color_unpack_ia4 (__u32 X);
-inline __u32 color_unpack_ia8 (__u32 X);
+__u32 color_unpack_rgb565 (__u32 X);
+__u32 color_unpack_rgba4 (__u32 X);
+__u32 color_unpack_rgba6 (__u32 X);
+__u32 color_unpack_rgb555 (__u32 X);
+__u32 color_unpack_rgb4a3 (__u32 X);
+__u32 color_unpack_rgb5a3 (__u32 X);
+__u32 color_unpack_i4 (__u32 X);
+__u32 color_unpack_i8 (__u32 X);
+__u32 color_unpack_ia4 (__u32 X);
+__u32 color_unpack_ia8 (__u32 X);
 
 unsigned int gx_draw (__u32 mem, int prim, int n, int vat);
 
diff -up 0.4/tplx.c~ 0.4/tplx.c
--- 0.4/tplx.c~	2004-12-27 16:57:42.000000000 +0100
+++ 0.4/tplx.c	2015-05-16 12:16:40.914146665 +0200
@@ -117,7 +117,7 @@ typedef struct
 __u8 texbuff[1024*1024*4];
 
 
-inline __u32 color_unpack_rgb565 (__u32 X)
+static inline __u32 color_unpack_rgb565 (__u32 X)
 {
 #ifdef LIL_ENDIAN
 	return (((X & 0xf800) >>  8)  | ((X & 0xe000) >> 13)  |
@@ -133,7 +133,7 @@ inline __u32 color_unpack_rgb565 (__u32
 }
 
 
-inline __u32 color_unpack_rgba4 (__u32 X)
+static inline __u32 color_unpack_rgba4 (__u32 X)
 {
 	__u32 _X = X;
 
@@ -147,7 +147,7 @@ inline __u32 color_unpack_rgba4 (__u32 X
 }
 
 
-inline __u32 color_unpack_rgba6 (__u32 X)
+static inline __u32 color_unpack_rgba6 (__u32 X)
 {
 #ifdef LIL_ENDIAN
 	return ((X & 0xfc0000) >> 16) | ((X & 0xc00000) >> 22) |
@@ -163,7 +163,7 @@ inline __u32 color_unpack_rgba6 (__u32 X
 }
 
 // RGB5 1rrr rrgg gggb bbbb
-inline __u32 color_unpack_rgb555 (__u32 X)
+static inline __u32 color_unpack_rgb555 (__u32 X)
 {
 #ifdef LIL_ENDIAN
 	return ((X & 0x7c00) >>  7) | ((X & 0x7000) >> 12) |
@@ -178,7 +178,7 @@ inline __u32 color_unpack_rgb555 (__u32
 
 
 // RGB4A3  0aaa rrrr gggg bbbb
-inline __u32 color_unpack_rgb4a3 (__u32 X)
+static inline __u32 color_unpack_rgb4a3 (__u32 X)
 {
 #ifdef LIL_ENDIAN
 	return ((X & 0x0f00) >>  4) | ((X & 0x0f00) >>  8) |
@@ -194,7 +194,7 @@ inline __u32 color_unpack_rgb4a3 (__u32
 }
 
 
-inline __u32 color_unpack_rgb5a3 (__u32 X)
+static inline __u32 color_unpack_rgb5a3 (__u32 X)
 {
 	if (X & 0x8000)
 		return (color_unpack_rgb555 (X) | MASK_ALPHA);
@@ -203,7 +203,7 @@ inline __u32 color_unpack_rgb5a3 (__u32
 }
 
 
-inline __u32 color_unpack_i4 (__u32 X)
+static inline __u32 color_unpack_i4 (__u32 X)
 {
 	X |= X << 4;
 	X |= X << 8;
@@ -211,14 +211,14 @@ inline __u32 color_unpack_i4 (__u32 X)
 }
 
 
-inline __u32 color_unpack_i8 (__u32 X)
+static inline __u32 color_unpack_i8 (__u32 X)
 {
 	X |= X << 8;
 	return (X | (X << 16));
 }
 
 
-inline __u32 color_unpack_ia4 (__u32 X)
+static inline __u32 color_unpack_ia4 (__u32 X)
 {
 	__u32 a = X & 0xf0;
 
@@ -234,7 +234,7 @@ inline __u32 color_unpack_ia4 (__u32 X)
 }
 
 
-inline __u32 color_unpack_ia8 (__u32 X)
+static inline __u32 color_unpack_ia8 (__u32 X)
 {
 	__u32 a = X & 0xff00;
 
diff -up 0.4/general.c~ 0.4/general.c
--- 0.4/general.c~	2005-01-12 01:21:50.000000000 +0100
+++ 0.4/general.c	2015-05-16 12:21:28.888736494 +0200
@@ -187,7 +187,7 @@ __u32 round_up (__u32 a, __u32 b)
 }
 
 
-inline int is_power_of_two (__u32 a)
+int is_power_of_two (__u32 a)
 {
 	int i = 0;
 
diff -up 0.4/general.h~ 0.4/general.h
--- 0.4/general.h~	2015-05-16 12:19:56.000000000 +0200
+++ 0.4/general.h	2015-05-16 12:21:07.481287198 +0200
@@ -82,7 +82,7 @@ char *kill_extension (char *filename);
 char *get_home_dir (void);
 
 __u32 round_up (__u32 a, __u32 b);
-inline int is_power_of_two (__u32 a);
+int is_power_of_two (__u32 a);
 
 __u32 crc_setup (unsigned int bits);
 __u32 crc_iterate (__u32 crc, __u8 d);
diff -up 0.4/mem.h~ 0.4/mem.h
--- 0.4/mem.h~	2004-12-27 16:53:42.000000000 +0100
+++ 0.4/mem.h	2015-05-16 12:20:44.666874096 +0200
@@ -218,8 +218,8 @@ void write_word_r (__u32 addr, __u32 dat
 void write_word (__u32 addr, __u32 data);
 void write_double (__u32 addr, __u64 data);
 
-inline __u8 mem_read8_safe (__u32 addr);
-inline __u32 mem_read32_safe (__u32 addr);
+__u8 mem_read8_safe (__u32 addr);
+__u32 mem_read32_safe (__u32 addr);
 
 void mem_fake_w32 (__u32 addr, __u32 data);
 __u32 mem_fake_r32 (__u32 addr);
